# HG changeset patch
# User Matthew Wild <mwild1@gmail.com>
# Date 1702388474 0
# Node ID 6877786d73d7e2b7f6cd6345560d03e0b6826fdf
# Parent  626ab0af83af942d0da824a3926b55ca5e418275
mod_storage_internal, tests: Fix before/after combined with the 'reverse' flag

diff -r 626ab0af83af -r 6877786d73d7 plugins/mod_storage_internal.lua
--- a/mod_storage_internal.lua	Sat Dec 09 21:01:49 2023 +0100
+++ b/mod_storage_internal.lua	Tue Dec 12 13:41:14 2023 +0000
@@ -180,6 +180,7 @@
 				i = i - 1
 				return list[i]
 			end
+			query.before, query.after = query.after, query.before;
 		end
 		if query.key then
 			iter = it.filter(function(item)
